apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -56,14 +64,33 @@ const initialFormState = {
   lastName: null,
   id: null,
   nationality: null,
+  date: {
+    day: '',
+    month: '',
+    year: '',
+  },
   dateOfBirth: null,
```

Comment:
Do we need the dateOfBirth state? On first look, it looks like duplicity with the date state.

Reply:
Not really anymore, thanks!

After removing it, had to change `State` a little bit, as couldnt find a way to reuse `PassengerType` without date of birth property, so I just copy pasted to the State and removed it

Do you know any way to avoid it? Was trying to use `$<Shape>` but didnt seem to work
Thanks!